ksql-rest-app/src/test/java/io/confluent/ksql/rest/entity/SourceDescriptionFactoryTest.java

```diff
@@ -83,7 +83,7 @@ public void tearDown() {
         schema,
         SerdeOption.none(),
         KeyField.of(schema.value().get(0).ref()),
-        new MetadataTimestampExtractionPolicy(),
+        Optional.empty(),
```

Comment:
Ditto, probably worth augmenting existing tests to test this works with a extractor policy